src/Valit/Rules/CollectionValitRule.cs

```diff
@@ -57,11 +57,16 @@ IValitResult IValitRule<TObject>.Validate(TObject @object)
             return result;
         }
 
-        private IValitResult ValidatePropertyRules(IEnumerable<IValitRule<TObject>> propertyRules, TObject @object)
-            => ValitRules<TObject>
-                .Create(propertyRules)
-                .WithStrategy(_strategy)
-                .For(@object)
-                .Validate();
+		private IValitResult ValidatePropertyRules(IEnumerable<IValitRule<TObject>> propertyRules, TObject @object)
+			=> ValitRules<TObject>
```

Comment:
Same as above, these are tabs instead of spaces.
@GooRiOn Are you OK with checking these 'problems' during a PR?